premium-app-example-wizard/config.yml

```diff
@@ -6,156 +6,136 @@ categories:
 steps:
 - title: Introduction
   content: |
-    This tutorial walks through the steps of implementing an installation flow that will configure the PureCloud environment for use by the Premium App.
+    The wizard is a simple web app built to provision additional items in PureCloud for use by the Premium App. This tutorial explores the sample Premium App Wizard installer and how partners can leverage it for their own Premium Apps. 
 
-    Common configurations that need to be done includes creating additional roles, creating groups, and creating additional instances of the app.
+    The four most provisioned PureCloud items are:
```

Comment:
While you're updating it, replace all instances of `PureCloud` with `Genesys Cloud`. This doesn't apply to literal values like classes or package names though.